packages/ilios-common/app/styles/ilios-common/components/weekly-calendar.scss

```diff
@@ -24,7 +24,7 @@
     grid-column: 1;
     grid-row: 1;
     justify-self: center;
-    margin: 0 0 0.25rem var(--hour-space);
```

Comment:
I don't see anywhere else this variable is being used. Can `--hour-space` go away?

Reply:
hmm, i see it used elsewhere in this component stylesheet

```
$ grep -r '\--hour-space' packages/ | egrep -v '(dist|node_modules)'
packages/ilios-common/app/styles/ilios-common/components/weekly-calendar.scss:  --hour-space: 2.9rem;
packages/ilios-common/app/styles/ilios-common/components/weekly-calendar.scss:    --hour-space: 5.5rem;
packages/ilios-common/app/styles/ilios-common/components/weekly-calendar.scss:    grid-template-columns: var(--hour-space) repeat(7, minmax(0, 1fr));
packages/ilios-common/app/styles/ilios-common/components/weekly-calendar.scss:    grid-template-columns: var(--hour-space) repeat(7, minmax(0, 1fr));
packages/ilios-common/app/styles/ilios-common/components/daily-calendar.scss:  --hour-space: 2.9rem;
packages/ilios-common/app/styles/ilios-common/components/daily-calendar.scss:    --hour-space: 5.5rem;
packages/ilios-common/app/styles/ilios-common/components/daily-calendar.scss:    margin: 0 0 0.25rem var(--hour-space);
packages/ilios-common/app/styles/ilios-common/components/daily-calendar.scss:    grid-template-columns: var(--hour-space) minmax(0, 1fr);
```

Reply:
looked a bit closer, and that variable is in use in that component stylesheet.

we'll have to keep it for now.

Reply:
Damn. I must have searched with my thumbs. Ugh. Sorry.